src/late_multicellular_stage/MetaballLayout.cs

```diff
@@ -209,4 +210,26 @@ public IEnumerable<T> GetMetaballsNotTouchingParents(float contactThreshold = 0.
             }
         }
     }
+
+    public Metaball? GetClosestMetaballToPosition(Vector3 position)
+    {
+        var closestDistance = float.MaxValue;
+        Metaball? closestMetaball = null;
+
+        foreach (var metaball in this)
+        {
+            if (metaball.Parent == null)
```

Comment:
I have one question, why cannot this return the root metaball? Sometimes that is closest to the specified position, right?

Reply:
You're right